orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -324,6 +325,21 @@ def _get_exported_model(self, input_schema: _ModelInputOutputSchemaType, *inputs
         # FlattenedModule needs _InputInfo to expand user input from *args to *args + **kwargs
         self._flattened_module._input_info = self._input_info
 
+        # Leverage cached model if available
+        cache_dir = self._runtime_options.ortmodule_cache_dir
+        rank = torch.distributed.get_rank() if torch.distributed.is_initialized() else 0
+        if cache_dir:
+            self._logger.info("ORTModule cache optimization is ON.")
```

Comment:
Can we just log this one time when reading the env variable in the runtime configuration class?